src/js/app.js

```diff
@@ -354,8 +354,11 @@ export function load(event) {
      * move this to the MAIN MENU
      */
     filename = files[0].name.split('.').slice(0, -1).join('.')
-    if (filename == '')
-      filename = files[0].name
+    if (filename == '') filename = files[0].name
+
+    // Update page title and menu filename
+    document.title = `${filename}`
```

Comment:
A suggestion to keep the initial name of the website in the title, making the browser tab recognizable:

```suggestion
    document.title = `${filename}` - DCML Reductive Analysis Webapp / Redusic / MuseReduce
```

Happy work on that project. ✌️